app/components/conversations/conversation-part-closed.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get } from '@ember/object';
+import { alias } from '@ember/object/computed';
+
+export default Component.extend({
+
+  author: null,
+
+  classNames: ['conversation-part-closed'],
+  classNameBindings: ['isSelf:conversation-part-closed--is-self'],
```

Comment:
Is this needed? I'm not sure what would differ stylistically here so it looks like it's just extra work.

Reply:
Do you mean  `classNameBindings: ['isSelf:conversation-part-closed--is-self'],`

Reply:
Right, I'm not sure why it's needed here, since there's nothing special about you vs someone else closing (vs the chat bubble where it's gray vs blue).